paystell-frontend/src/app/PaymentLinkScreen/page.tsx

```diff
@@ -0,0 +1,131 @@
+import * as Dialog from '@radix-ui/react-dialog';
+import Image from 'next/image';
+import Logo_PayStell from './Images/Logo_PayStell.png';
+import Logo_Company from './Images/Logo_Company.png';
+import Logo_PayStell_P from './Images/Logo_PayStell_P.png';
+import Product from './Images/Product.png';
+
```

Comment:
add import component
```suggestion
import { Button } from "@/components/ui/button";

```